### DISCUSSION TITLE
Slope calibration for libre patched app source

### DISCUSSION THREAD

**Discussion by enpa23** [NightscoutFoundation/xDrip#2771] [General]
With FSL3 xDrip could be used only with Juggluco as data source. This means to use libre patched app protocol.
Calibrations on patched app data source are affecting the intercept only, which makes them not useful nor reliable to be used.
As a workaround, secondary plugins ( last7Ua or Datricsae)  have to enabled.  ( even with the secondary plugin enabled, on the main graph the reported slope and intercept are those of the default one ) 
![image](https://user-images.githubusercontent.com/45668841/230633133-a7cd81a6-b734-484a-9d98-9daea1c28ff6.png)
![image](https://user-images.githubusercontent.com/45668841/230633179-a9228a80-632d-4d28-b088-85afd743618e.png)

Secondary plugins are flagged as experimental and need lot of calibrations to work ( at least 4 for Datricsae and 7 for Last7Ua ) otherwhise xDrip switches to the default algorithm.
Given the importance of calibration even with FSL3 and the limits of the current workaround, is there any plan to enable slope calibration for the default plugin?

**Comment by Navid200** [comment]
@tzachi-dar
